react/src/app.jsx

```diff
@@ -1,28 +1,27 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
 import { OverallRatings } from './overallRatings.jsx';
-import { Review } from './review.jsx';
+import { ReviewsList } from './reviewsList.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       data: []
     }
-
-    this.handleClick = this.handleClick.bind(this);
   }
```

Comment:
small nitpick, but indentation is off starting here.  Made me think your `fetch` method was outside of `App` instead of inside it.